Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -163,60 +160,65 @@ class TextureCacheBase
   virtual bool CompileShaders() = 0;
   virtual void DeleteShaders() = 0;
 
-  static TCacheEntryBase* Load(const u32 stage);
-  static void UnbindTextures();
+  TCacheEntryBase* Load(const u32 stage);
+  void UnbindTextures();
   virtual void BindTextures();
-  static void CopyRenderTargetToTexture(u32 dstAddr, unsigned int dstFormat, u32 dstStride,
-                                        PEControl::PixelFormat srcFormat,
-                                        const EFBRectangle& srcRect, bool isIntensity,
-                                        bool scaleByHalf);
+  void CopyRenderTargetToTexture(u32 dstAddr, unsigned int dstFormat, u32 dstStride,
+                                 PEControl::PixelFormat srcFormat, const EFBRectangle& srcRect,
+                                 bool isIntensity, bool scaleByHalf);
 
   virtual void ConvertTexture(TCacheEntryBase* entry, TCacheEntryBase* unconverted, void* palette,
                               TlutFormat format) = 0;
 
 protected:
   TextureCacheBase();
 
-  alignas(16) static u8* temp;
-  static size_t temp_size;
+  alignas(16) u8* temp = nullptr;
```

Comment:
Does the member containing the pointer need to be aligned here? If I'm understanding things correctly, this pointer itself is aligned (AllocateAlignedMemory), then loaded and passed to the texture decoder, so the address of temp isn't used with any SSE instructions.